microshift.sh

```diff
@@ -74,7 +74,7 @@ function download_microshift_rpm {
     elif [ -n "${MICROSHIFT_NVR-}" ]; then
         nvr_suffix="-${MICROSHIFT_NVR-}"
     fi
-    sudo yum download ${extra_opts} --downloaddir ${pkgDir} --downloadonly microshift${nvr_suffix} microshift-networking${nvr_suffix} microshift-release-info${nvr_suffix} microshift-selinux${nvr_suffix} microshift-greenboot${nvr_suffix}
+    sudo yum download ${extra_opts} --downloaddir ${pkgDir} --downloadonly microshift${nvr_suffix} microshift-networking${nvr_suffix} microshift-release-info${nvr_suffix} microshift-selinux${nvr_suffix} microshift-greenboot${nvr_suffix} microshift-olm${nvr_suffix}
```

Comment:
The `microshift-olm` RPM is apparently optional now?
> To simplify the use of operators, we enabled the operator lifecycle manager (OLM) as an optional component for MicroShift (“dnf install microshift-olm”).

Reply:
@cfergeau yes but I think for crc point of view better to have it default. So user doesn't need to manually install it if required since it is still under the resource limit.